server/app/routes/games.router.js

```diff
@@ -37,9 +36,19 @@ router.get('/:id', function(req, res, next){
 })
 
 router.post('/', function(req, res, next){
```

Comment:
The logic in this route handler is getting a little long, and it's nice to keep route handler code short and simple. Consider maybe writing a class or instance method on the model—e.g. `Game.createWithAllAssociations` or something.